Packages/com.chisel.editor/Chisel/Editor/ComponentEditors/Base/ChiselNodeEditor.cs

```diff
@@ -356,7 +356,7 @@ protected void ShowSurfaces(SerializedProperty surfacesProp, GUIContent[] kSurfa
         protected abstract void InitInspector();
 
         protected abstract void OnInspector();
-        protected virtual bool OnGeneratorValidate(T generator) { return generator.isActiveAndEnabled; }
+        protected virtual bool OnGeneratorValidate(T generator) { return generator != null && generator.isActiveAndEnabled; }
```

Comment:
where how/did this get triggered?

Reply:
I suspect it was some combination of having a half-complete brush constructed in the editor, reloading scripts, and using Ctrl-Z judiciously.

I'm not able to replicate the behavior right now, so I'm going to undo this change in-case it's just masking an underlying issue...